sql_metadata/parser.py

```diff
@@ -714,6 +729,37 @@ def _resolve_alias_to_column(self, alias_token: SQLToken) -> str:
             alias_of = alias_token.left_expanded
         return alias_of
 
+    def _resolve_subquery_alias_to_column(self, subquery_alias) -> List[str]:
+        """
+        Resolves subquery reference to the actual column in the subquery
+        """
+        parts = subquery_alias.split(".")
+        if len(parts) != 2 or parts[0] not in self.subqueries_names:
+            return [subquery_alias]
+        sub_qry, column_name = parts[0], parts[-1]
+        sub_qry_definition = self.subqueries.get(sub_qry)
+        subparser = self._subqueries_parsers.setdefault(
+            sub_qry, Parser(sub_qry_definition)
```

Comment:
Can we use here `sub_query` instead of a short name (`sub_qry`) that only save us two letters?

Reply:
Sure, will update

Reply:
Done